src/lib/route-broadcaster.js

```diff
@@ -27,6 +27,7 @@ class RouteBroadcaster {
     this.routeCleanupInterval = config.routeCleanupInterval
     this.routeBroadcastInterval = config.routeBroadcastInterval
     this.routingTables = routingTables
+    if (this.routingTables.publicTables.currentEpoch !== 0) throw new Error('expecting a fresh routingTables with epoch support')
```

Comment:
It was originally a way of making sure I was getting all the entry points into the related code.  Now, it serves to error out early if you try to use an incompatible ilp-routing, which has been helpful to me.  Its use may be served now though.